Source/ACE.Server/WorldObjects/Managers/EmoteManager.cs

```diff
@@ -1342,7 +1340,12 @@ public void Enqueue(BiotaPropertiesEmote emoteSet, WorldObject targetObject, int
                         Nested--;
 
                         if (Nested == 0)
+                        {
                             IsBusy = false;
+                            if (OnDeathEmoteInProgress)
```

Comment:
this has a chance to fail, and OnDeathEmoteInProgress to remain true forever

as mentioned, IsBusy and Nested state can get trashed with simultaneous emotes running, and the state of these 2 vars cannot be trusted reliably after OnDeath sets IsBusy=false and forces itself to run.

Reply:
update: OnDeathEmoteInProgress will not remain true forever as originally thought, it just might be delayed if a Nested emote is already running when OnDeath is called. very unlikely / not a realistic scenario with any emote data i'm aware of, so not a big issue